adapters/cwire/cwiretest/supplemental/badresponse.json

```diff
@@ -0,0 +1,89 @@
+{
+  "mockBidRequest": {
+    "id": "80ce30c53c16e6ede735f123ef6e32361bfc7b22",
+    "cur": [
+      "CHF"
+    ],
+    "imp": [
+      {
+        "id": "102",
+        "banner": {
+          "h": 250,
+          "w": 300,
+          "pos": 0
+        }
+      }
+    ],
+    "site": {
+      "page": "http://www.foobar.com/1234.html"
+    },
+    "device": {
+      "ua": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:108.0) Gecko/20100101 Firefox/108.0"
+    },
+    "user": {
+      "id": "55816b39711f9b5acf3b90e313ed29e51665623f",
+      "geo": {
+        "country": "ch",
+        "region": "basel-stadt",
+        "city": "basel"
+      }
+    }
+  },
+  "httpCalls": [
+    {
+      "expectedRequest": {
+        "uri": "https://cwi.re/prebid/adapter-endpoint",
+        "headers": {
+          "Content-Type": [
+            "application/json;charset=utf-8"
+          ],
+          "Accept": [
+            "application/json"
+          ]
+        },
+        "body": {
+          "id": "80ce30c53c16e6ede735f123ef6e32361bfc7b22",
+          "cur": [
+            "CHF"
+          ],
+          "imp": [
+            {
+              "id": "102",
+              "banner": {
+                "h": 250,
+                "w": 300,
+                "pos": 0
+              }
+            }
+          ],
+          "site": {
+            "page": "http://www.foobar.com/1234.html"
+          },
+          "device": {
+            "ua": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:108.0) Gecko/20100101 Firefox/108.0"
+          },
+          "user": {
+            "id": "55816b39711f9b5acf3b90e313ed29e51665623f",
+            "geo": {
+              "country": "ch",
+              "region": "basel-stadt",
+              "city": "basel"
+            }
+          }
+        }
+      },
+      "mockResponse": {
+        "status": 200,
+        "body": {
+          "id": 3
+        }
+      }
+    }
+  ],
+  "expectedMakeBidsErrors": [
+    {
+      "value": "Error while decoding response, err: .*",
+      "comparison": "regex"
+    }
+  ]
```

Comment:
With the added check for any other non-200 responses, can you please also add a JSON test for a non-200 response that's not 204?